### PR TITLE
Bug 1834454 - Webhooks should fire if bug removed from matching product

### PR DESCRIPTION
**Pull request by justdave**
See [bug 1834454](https://bugzilla.mozilla.org/show_bug.cgi?id=1834454)

Caveat: I have no way to test this.

### CODE REVIEW

File: extensions/Push/lib/Connector/Webhook.pm [hunk 1]
```diff
@@ -84,6 +84,32 @@ sub should_send {
...
+          $removed_product = $change->{'removed'};
+        }
+        if ($change->{'field'} eq 'component') {
+          $removed_component = $change->{'removed'};
+        }
+      }
+    }
+    if ($removed_product || $removed_component) {
+      if ($removed_product eq '') {
+        $removed_product eq $bug_data->{'product'};
```
Comment (dklawren) [inline extensions/Push/lib/Connector/Webhook.pm:103]: $removed_product = $bug_data->{'product'};

**Review by dklawren — CHANGES_REQUESTED** [review]
Looks good except for one change.

File: extensions/Push/lib/Connector/Webhook.pm (hunk 1 above)
Comment (justdave) [reply extensions/Push/lib/Connector/Webhook.pm:103]: Bwahahaha, I can't believe I did that.

Comment (justdave) [reply extensions/Push/lib/Connector/Webhook.pm]: Ok, fixed.

**Review by dklawren — APPROVED** [review]

**Pull request merged**